<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitsComboBoxTest</class>
 <widget class="QWidget" name="UnitsComboBoxTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unit Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="unitsWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Time Format:</string>
        </property>
        <property name="buddy">
         <cstring>timeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="timeCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="geodeticLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Geodetic Units:</string>
        </property>
        <property name="buddy">
         <cstring>geodeticCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="geodeticCombo">
        <property name="toolTip">
         <string>Units for geodetic (LLA) angle text</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="distanceLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Distance Units:</string>
        </property>
        <property name="buddy">
         <cstring>distanceCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="distanceCombo">
        <property name="toolTip">
         <string>Units for distance text</string>
        </property>
        <property name="maxVisibleItems">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="altitudeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Altitude Units:</string>
        </property>
        <property name="buddy">
         <cstring>altitudeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="altitudeCombo">
        <property name="toolTip">
         <string>Units for altitude text</string>
        </property>
        <property name="maxVisibleItems">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="angleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Angle Units:</string>
        </property>
        <property name="buddy">
         <cstring>angleCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="angleCombo">
        <property name="toolTip">
         <string>Units for angle text</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="speedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Speed Units:</string>
        </property>
        <property name="buddy">
         <cstring>speedCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="speedCombo">
        <property name="toolTip">
         <string>Units for speed text</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="coordLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Coordinate System:</string>
        </property>
        <property name="buddy">
         <cstring>coordCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="coordCombo">
        <property name="toolTip">
         <string>Coordinate system for position text</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="magVarLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Magnetic Variance:</string>
        </property>
        <property name="buddy">
         <cstring>magVarCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="magVarCombo">
        <property name="toolTip">
         <string>Magnetic variance for angle text</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="vertDatumLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Vertical Datum:</string>
        </property>
        <property name="buddy">
         <cstring>vertDatumCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="vertDatumCombo">
        <property name="toolTip">
         <string>Vertical datum for altitude text</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="inputValueSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>-99999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="angleLabel_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>meters =</string>
       </property>
       <property name="buddy">
        <cstring>angleCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>timeCombo</tabstop>
  <tabstop>geodeticCombo</tabstop>
  <tabstop>distanceCombo</tabstop>
  <tabstop>altitudeCombo</tabstop>
  <tabstop>angleCombo</tabstop>
  <tabstop>speedCombo</tabstop>
  <tabstop>coordCombo</tabstop>
  <tabstop>magVarCombo</tabstop>
  <tabstop>vertDatumCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
